.github/codecov.yml

```diff
@@ -24,4 +24,4 @@ comment:
 
 ignore:
   - "tests/zfs-tests"		# Don't need Tests to cover themselves
-  - "contrib/zstd"		# Dependencies don't need coverage testing
+  - "contrib/zstd/**/*"		# Dependencies don't need coverage testing
```

Comment:
interesting, what's the difference to zfs-tests here?

Reply:
"contrib/zstd" is simply not the right syntax for recursiveness.

Reply:
so previous version did nothing >.<